styles/common.css

```diff
@@ -1,17 +1,71 @@
+html, body {
+    margin: 0;
+    padding: 0;
+}
+
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    background-color: #DCA47C;
+}
+
+
 .wrapper {
     display: flex;
     width: 100%;
-    height: 100vh;
-    background: lavender;
+    background: #FCF8F3;
+    justify-content: center;
+}
+
+
+.content {
+    max-width: fit-content;
+}
+
+
+.page-header {
+    display: flex;
     justify-content: space-between;
+    align-items: center;
 }
 
-.sidebar {
-    width: 30%;
-    background: beige;
+
+h1 {
+    font-size: 30pt;
+    font-family: "Roboto Slab", serif;
+    font-optical-sizing: auto;
+    font-weight: 600;
+    font-style: normal;
 }
 
-.content {
-    width: 60%;
-    background: beige;
-}
\ No newline at end of file
+/* Button styling */
+button {
+    border: none;
+    background-color: transparent;
+}
+
+.home-button {
+    align-content: center;
+    margin-left: 30px;
+}
+
+div.menu-items > a > button {
```

Comment:
this is not really good. 
You made too specific selector. As a result, it could be hard to override someday in the future.
Take a look here: https://dev.to/smitterhane/how-to-calculate-css-specificity-of-your-style-rules-moj

instead it's better to provide a class name: "menu-items-button" for example

Reply:
The buttons now have the class "navbar-button" and are selected based on class in common.css